package.json

```diff
@@ -11,6 +11,9 @@
     "micro-services"
   ],
   "dependencies": {
+    "async": "^2.6.0",
```

Comment:
isn't mocha missing in devDependencies and maybe even scripts.test?

Reply:
yes, added!